File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/AuxInput.scala

package org.openapitools.client.model


case class AuxInput (
    /* Timestamp in Unix epoch milliseconds. */
    _timeMs: Any,
    /* Boolean representing the digital value of the aux input. */
    _value: Boolean
)
object AuxInput {
    def toStringBody(var_timeMs: Object, var_value: Object) =
        s"""
        | {
        | "timeMs":$var_timeMs,"value":$var_value
        | }
        """.stripMargin
}

File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/AuxInput.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * Digital value of an aux input.
 * @param timeMs Timestamp in Unix epoch milliseconds.
 * @param value Boolean representing the digital value of the aux input.
 */
data class AuxInput (

        @get:NotNull 
        @JsonProperty("timeMs") val timeMs: Any,

        @get:NotNull 
        @JsonProperty("value") val value: Boolean
) {

}

